core/App/screens/PinCreate.tsx

```diff
@@ -127,21 +139,42 @@ const PinCreate: React.FC<PinCreateProps> = ({ setAuthenticated }) => {
         </Text>
         <PinInput
           label={t('PinCreate.EnterPINTitle')}
-          onPinChanged={setPin}
+          onPinChanged={(p: string) => {
+            setPin(p)
+
+            if (p.length === minPINLength) {
+              if (pinTwoInputRef && pinTwoInputRef.current) {
+                pinTwoInputRef.current.focus()
+                const reactTag = findNodeHandle(pinTwoInputRef.current)
+                if (reactTag) {
+                  AccessibilityInfo.setAccessibilityFocus(reactTag)
+                }
+              }
+            }
+          }}
           testID={testIdWithKey('EnterPIN')}
           accessibilityLabel={t('PinCreate.EnterPIN')}
-          autoFocus={true}
+          autoFocus={false}
         />
         <PinInput
           label={t('PinCreate.ReenterPIN')}
           onPinChanged={(p: string) => {
             setPinTwo(p)
+
             if (p.length === minPINLength) {
               Keyboard.dismiss()
+              if (createPinButtonRef && createPinButtonRef.current) {
+                const reactTag = findNodeHandle(createPinButtonRef.current)
```

Comment:
Might want to keep a note that this will be deprecated in React 18: https://reactnative.dev/docs/new-architecture-library-intro#preparing-your-javascript-codebase-for-the-new-react-native-renderer-fabric

Reply:
Note added.